lib/csv_query/database.rb

```diff
@@ -1,37 +1,62 @@
 require 'sqlite3'
+require 'csv_query/table'
 
 module CsvQuery
-  # Wraps a SQLite in-memory database with a single table named csv.
+  # Wraps a SQLite in-memory database with a single table named csv by default if you pass in only a csv to initialize.
+  # Don't pass in a CSV and use #create_table and #import_data_from_csv to to perform more complex multi-table loads.
```

Comment:
These comments look like they are documenting the use of `#initialize` and should probably be moved there, I guess. Comments here document the purporse of the class. If it is the "single table" part that bothers you, I'd be fine removing that part.